extension/src/popup/components/WarningMessages/styles.scss

```diff
@@ -418,14 +332,23 @@
     display: flex;
     justify-content: center;
     margin-bottom: 0.5rem;
+
+    .AccountAssets__asset--logo {
+      margin-right: 0;
+    }
   }
 
   &__code,
   &__domain {
     display: flex;
+    font-size: pxToRem(14px);
     justify-content: center;
   }
 
+  &__domain {
+    color: var(--sds-clr-gray-11);
```

Comment:
there was some discrepancies from the Figma in the styling for these warnings, so addressing while I'm here